### PR TITLE
Added options from Maseya Palette Randomizer

### PR DESCRIPTION
**Pull request by Edos512**

### CODE REVIEW

**Comment by Edos512** [comment]
Added options for other palette shuffling modes from maseya.

Link palette disabled as it does not work properly with random sprite on event.

When adjusting seeds colors are first restored to the original using the vanilla rom that resides in the directory.

File: Rom.py [hunk 1]
```diff
@@ -37,15 +37,19 @@
 
 class LocalRom(object):
 
-    def __init__(self, file, patch=True, name=None, hash=None):
+    def __init__(self, file, patch=True, vanillaRom=None, name=None, hash=None):
         self.name = name
         self.hash = hash
         self.orig_buffer = None
+        self.vanillaBuffer = None
```
Comment (Berserker66) [inline Rom.py:44]: orig_buffer is already the "original rom", aka vanilla
Let's not duplicate code and data.

Comment (Edos512) [reply Rom.py:44]: Then i have to load the vanilla rom into orig_buffer instead of vanillaBuffer and that's it

**Comment by Berserker66** [comment]
Thank you very much

**Pull request merged**